src/Common/RandomNumberManagerProduction.cpp

```diff
@@ -0,0 +1,36 @@
+/*****************************************************************************
+ * Copyright (c) 2018-2022 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#include "RandomNumberManagerProduction.h"
+#include <ctime>
+
+using namespace openblack;
+
+bool RandomNumberManagerProduction::SetSeed(int seed)
+{
+	generator().seed(seed);
+	return true;
+}
```

Comment:
Do we need to manually set seed in production?

Reply:
In that case wouldn't that be another implementation?
Maybe add a MultiplayerSynced implementation